packages/react-native/src/utils/add-jest.ts

```diff
@@ -32,9 +32,13 @@ export async function addJest(
   resolver: '@nx/jest/plugins/resolver',
   moduleFileExtensions: ['ts', 'js', 'html', 'tsx', 'jsx'],
   setupFilesAfterEnv: ['<rootDir>/test-setup.${js ? 'js' : 'ts'}'],
+  transformIgnorePatterns: ['node_modules/(?!react-native|jest-runner)/'],
```

Comment:
by default, react native jest preset already has a transformIgnorePatterns like:
https://github.com/facebook/react-native/blob/main/packages/react-native/jest-preset.js#L24
   ` 'node_modules/(?!((jest-)?react-native|@react-native(-community)?)/)',`
is this still needed?

Reply:
in that case probably not. I assume it's just extra deps that were added that ended up needing the patterns without realize it. I'll remove. 👍

Reply:
@barbados-clemens Oups, i'm a bit late !
This is still needed apparently.
Easy to reproduce: create a bare project, add lib, launch tests, and you'll get something like 

> Jest failed to parse a file. This happens e.g. when your code or its dependencies use non-standard JavaScript syntax, or when Jest is not configured to support such syntax.